modules/storage-aws-s3/s3/s3.go

```diff
@@ -56,7 +64,7 @@ func New(config Config, logger logrus.FieldLogger) (modulecapabilities.SnapshotS
 	if err != nil {
 		return nil, errors.Wrap(err, "create client")
 	}
-	return &s3{client, config, logger}, nil
+	return &s3{client, config, logger, config.BucketName()}, nil // Assuming that the active database files are always in the bucket root?
```

Comment:
here 4th argument is `dataPath` not `bucketName`. I think that `dataPath` this should be passed to `New(...)` so that then it could be used within `s3`